api/src/wsmanager.py

```
@@ -33,19 +40,16 @@ def start_server(self) -> None:
     ) -> None:
         room_id = room_id.lstrip('/')
         print(room_id)
-        while True:
-            try:
-                self._valid_room_ids.add(str(self.uuid_q.get_nowait()))
-            except queue.Empty:
-                break
-        if room_id in self._valid_room_ids:
+        if is_valid_uuid(room_id):
             response = self.gss.new_connection_request(client, room_id)
             await self.send_message_to_client(response, client)
             try:
                 async for message in client:
                     await self.consume(message, room_id, client)
             finally:
                 self.gss.connection_dropped(client, room_id)
+        else:
+            print(f'Invalid uuid: {room_id}')
```

Can you return an error here? Instead of just logging it